skulpin-renderer/src/renderer.rs

```diff
@@ -254,27 +254,23 @@ impl Renderer {
         //
         // Do skia drawing (including the user's callback)
         //
-        let mut canvas = self.skia_surface.as_mut().unwrap().surface.canvas();
+        let canvas = self.skia_surface.as_mut().unwrap().surface.canvas();
 
         let coordinate_system_helper = CoordinateSystemHelper::new(window_size, scale_factor);
 
         match self.coordinate_system {
             CoordinateSystem::None => {}
-            CoordinateSystem::Physical => {
-                coordinate_system_helper.use_physical_coordinates(&mut canvas)
-            }
-            CoordinateSystem::Logical => {
-                coordinate_system_helper.use_logical_coordinates(&mut canvas)
-            }
+            CoordinateSystem::Physical => coordinate_system_helper.use_physical_coordinates(canvas),
+            CoordinateSystem::Logical => coordinate_system_helper.use_logical_coordinates(canvas),
```

Comment:
Only changes are removing `&mut` but again `cargo fmt` complained about the block wrap. 🤷🏼‍♂️